src/index.js

```diff
@@ -1,8 +1,16 @@
 // Core
-export createRedux from './createRedux';
-export createDispatcher from './createDispatcher';
+import createRedux from './createRedux';
+import createDispatcher from './createDispatcher';
 
 // Utilities
-export composeMiddleware from './utils/composeMiddleware';
-export composeStores from './utils/composeStores';
-export bindActionCreators from './utils/bindActionCreators';
+import composeMiddleware from './utils/composeMiddleware';
```

Comment:
eslint was complaining about the import and export in the same statement. I had to split it up.